internal/db/request.go

```diff
@@ -53,10 +53,10 @@ func (db *db) execRequest(ctx context.Context, request string) *client.RequestRe
 	results, err := planner.RunRequest(ctx, parsedRequest)
 	if err != nil {
 		res.GQL.Errors = []error{err}
-		return res
 	}
-
-	res.GQL.Data = results
+	if len(results) > 0 {
+		res.GQL.Data = results[0]
```

Comment:
question/suggestion: Why is results still an array?  Can we return just a `map[string]any` now instead?

Same question/suggestion for `RunSelection`.

Reply:
Good catch! I've updated both to return maps.